tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -527,3 +527,20 @@ class DebugProtocolPart(ProtocolPart):
     def load_devtools(self):
         """Load devtools in the current window"""
         pass
+
+    def load_reftest_analyzer(self, test, result):
```

Comment:
Yeah, because this part of the code is also shared with the WebDriver executor and is pretty generic; it only uses other parts of the protocol so as long as you implement those (and it's really only "load page" and "execute script") you can get this without a custom implementation. Of course you still need to call it in the executor at the right time (and ensure the relevant window is active).

Reply:
🤦 

I totally misread the code, and thought that `WebDriverDebugProtocolPart` had an override of `load_reftest_analyzer` that raised. What it actually has is an override of `load_devtools` that raises. Sorry!